Added Deploy to Azure button to Azure Arc-enabled servers scenarios (#1248)
microsoft/azure_arc: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_defender/_index.md

    > **NOTE: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/azure/role-based-access-control/best-practices)**

## Deployment Options and Automation Flow

This Jumpstart scenario provides multiple paths for deploying and configuring resources. Deployment options include:

- Azure portal
- ARM template via Azure CLI

> For you to get familiar with the automation and deployment flow, below is an explanation.

Reviewer: ```suggestion
The steps below will help you get familiar with the automation and deployment flow.
```

> - User is providing the ARM template parameters values, either via the portal or editing the parameters file (1st-time edit). These parameters values are being used throughout the deployment.

Reviewer: ```suggestion
- User provides the ARM template parameter values, either via the portal or editing the parameters file. These parameters values are used throughout the deployment.
```
